micrometer-core/src/main/java/io/micrometer/core/instrument/binder/cache/JCacheMetrics.java

```diff
@@ -129,17 +129,19 @@ protected void bindImplementationSpecificMetrics(MeterRegistry registry) {
     }
 
     private Long lookupStatistic(String name) {
-        try {
-            List<MBeanServer> mBeanServers = MBeanServerFactory.findMBeanServer(null);
-            for (MBeanServer mBeanServer : mBeanServers) {
-                try {
-                    return (Long) mBeanServer.getAttribute(objectName, name);
-                } catch (AttributeNotFoundException | InstanceNotFoundException ex) {
-                    // did not find MBean, try the next server
+        if (objectName != null) {
```

Comment:
This guard looks important.

Reply:
Yes, even though probability is low there is still a chance of having `cacheManager` as null and it's better to be on the safe side.